eth/contracts/facets/DFGetterFacet.sol

```diff
@@ -315,91 +315,127 @@ contract DFGetterFacet is WithStorage {
         return ret;
     }
 
-    function getArtifactById(uint256 artifactId)
-        public
-        view
-        returns (ArtifactWithMetadata memory ret)
-    {
-        Artifact memory artifact = DFArtifactFacet(address(this)).getArtifact(artifactId);
+    function getArtifactActivationTimeOnPlanet(uint256 locationId) public view returns (uint256) {
+        return gs().planetArtifactActivationTime[locationId];
+    }
 
-        address owner;
+    // function getArtifactById(uint256 artifactId)
```

Comment:
We can just delete this, but might want to add it back bc of https://github.com/dfdao/game/pull/27